Bottom: 2209669edf35b42156d008a167fd733c633ff77a
Top:    c5ceb76922688ccc69387b32240d1c7baf973fa8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-03 12:13:06 +0000

Refresh of lesstif-silence-set-but-unused-1

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index cb3b01e..14a0695 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -3947,7 +3947,6 @@ lesstif_progress_dialog (int so_far, int total, const char *msg)
 static int
 lesstif_progress (int so_far, int total, const char *message)
 {
-  static bool visible = false;
   static bool started = false;
   XEvent e;
   struct timeval time;
@@ -3958,7 +3957,6 @@ lesstif_progress (int so_far, int total, const char *message)
   if (so_far == 0 && total == 0 && message == NULL)
     {
       XtUnmanageChild (progress_dialog);
-      visible = false;
       started = false;
       progress_cancelled = false;
       return retval;
